zilencer/management/commands/sync_api_key.py

```diff
@@ -6,7 +6,11 @@
 from django.core.management.base import BaseCommand
 from zerver.models import get_user_profile_by_email, UserProfile
 import os
-from six.moves.configparser import SafeConfigParser
+import six
+if six.PY2:
+    from six.moves.configparser import SafeConfigParser
+else:
+    from configparser import ConfigParser
```

Comment:
I think we should potentially just use `from six.moves.configparser import ConfigParser` and use the Python 3 API everywhere, rather than needing the `six.PY2` checks...